Gemfile

```diff
@@ -16,10 +16,20 @@ gem 'bcrypt'
 gem 'fabrication'
 gem 'faker'
 
+gem "figaro"
+
+gem 'sidekiq'
+
+gem 'unicorn'
+gem 'rack-timeout'
+
+gem "sentry-raven", :git => "https://github.com/getsentry/raven-ruby.git"
```

Comment:
You could move this to `group :production` below, as thats where you are using it only.